src/Php55/composer.json

```diff
@@ -16,18 +16,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "ircmaxell/password-compat": "~1.0"
+        "php": ">=7.2"
     },
-    "autoload": {
-        "psr-4": { "Symfony\\Polyfill\\Php55\\": "" },
-        "files": [ "bootstrap.php" ]
+    "replace": {
+        "ircmaxell/password-compat": "*"
```

Comment:
I don't think we need to replace that package. We should just stop requiring it

Reply:
Since we guarantee the minimum php version, we can and should replace it to skip the package IMHO.

Reply:
Neither do I think that this is a good idea. This setting prevents the installation of `ircmaxell/password-compat` with no way around it. We don't own `ircmaxell/password-compat` and it shouldn't be up to us to decide if an application is allowed to install it or not.

Or let's put it that way:

* We just remove the dependency. Worst case: Some other library still requires it and we end up with an unnecessary package. Damage: zero.

* We declare that we replace the package. Worst case: An app or some other library relies on internals of that package (it shouldn't, but who am I to judge) and the app breaks.

I'd prefer to just remove `ircmaxell/password-compat` from the dependencies as @stof suggested.

Reply:
Fine, updated.